<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>member_category</class>
 <widget class="QWidget" name="member_category">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#label_name, #label_count {
	font-size: 10pt;
	font-weight: 600;
}

#label_count {
	padding-left: 6px;
	padding-right: 6px;
	background-color: #353535;
	border-radius: 9px;
}</string>
  </property>
  <layout class="QGridLayout" name="layout_widget" rowstretch="0,1" columnstretch="0,0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="1">
    <widget class="QLabel" name="label_count">
     <property name="text">
      <string notr="true">1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>Online</string>
     </property>
     <property name="indent">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <layout class="QVBoxLayout" name="layout_members">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
